src/index.js

```diff
@@ -55,6 +55,11 @@ export function app(state, actions, view, container) {
 
     var next = view(globalState, wiredActions)
     if (container && !renderLock) {
+      next = {
+        nodeName: container.nodeName.toLowerCase(),
+        attributes: {},
+        children: [].concat(next)
```

Comment:
or `next.pop /* Array */ ? next : [next]`

Reply:
Why not `Array.isArray(next)` (IE 9 compatible)?

Reply:
because of bytes and perfomance? see 4930b5ecd06d9d5c5d6fe7311769020d313fcd6c

Reply:
Size over correctness... is bad thing

Reply:
Then you might not like this repo...

Reply:
@chicoxyzzy That's a bit too dogmatic. We take a lighter approach. If we can get away with it, I suggest we do it. Not always, but if we can why not? :)